### DISCUSSION TITLE
BokehPivot Tool - Viewing Question

### DISCUSSION THREAD

**Discussion by tumibytes** [NREL/ReEDS-2.0#98] [Q&A: Data]
Hi, ReEDS Team! I wanted to try using the BokehPivot Module and follow the YouTube video you provide, but the format of my browser makes it difficult to use. I can't see the path I pasted unless I highlight it. Similarly, I cannot see if I selected "ReEDS 2.0" or "CSV" under "Type." Yesterday, I was not running into this problem. I just wanted to ask if there was a quick fix for this. What I am seeing is in the image below. Thank you!

![image](https://user-images.githubusercontent.com/97255496/155582823-8bf77b37-344c-4cc5-b476-14442d76c791.png)

**Comment by vcelloho** [comment]
Which web browser are you using to access the bokeh pivot viewer? You can copy the URL from one browser to another to switch if the default browser is causing issues.

**Reply by tumibytes** [reply]
I was using Google Chrome. I just used Microsoft Edge, and it helped with the viewing issue, thank you! However, I am still unable to see the "Model Variables," "Meta," "Filter Scenarios," "Build Report," and "Result" options. Is this another issue with the browser I am using?

What I see: 
![image](https://user-images.githubusercontent.com/97255496/155590627-23b2e5c9-a428-4fba-a9e1-b062a6b0f267.png)

What I am missing:
![image](https://user-images.githubusercontent.com/97255496/155590478-6e57d3f6-34e0-409e-b71f-dcffb046c541.png)

**Comment by vcelloho** [comment]
Chrome should work fine, it's what we use when working with bokeh pivot. In the dialog box that opens when the bokeh pivot batch file is executed what messages are appearing. There may be some helpful error messages letting us know what's wrong.

Also you can point the path towards a run or a folder of runs without using a csv file if that's what you're currently doing. COuld be another way to see where the error is.

**Reply by tumibytes** [reply]
I used a path to a folder of runs instead of a csv file, and it looks like it worked! Thank you again.

**Reply by vcelloho** [reply]
In that case there's likely an issue with the csv file that you tried to use, you could try debugging it. Feel free to post a screenshot I might be able to find something.
